proto/api/v1/messages.proto

```diff
@@ -35,6 +96,28 @@ message Scheduler {
   google.protobuf.Timestamp created_at = 6;
   // Max surge of rooms 
   string max_surge = 7;
+  // GameRoom spec
+  Spec spec = 8;
+}
+
+// Scheduler message used in the "ListScheduler version" definition. The "spec" is not implemented
+// on this message since it's unecessary for the list function
+message ListedScheduler {
```

Comment:
Appearlly this message is a structure to use like a list and not a list by itself.

normally the message are create to define a data structure and should not be associate with and action (verb), beside that is not a good practice to conjugate verb to to express an action. When you use like that you are breaking the semantics definitions. 

I think you could rename this message to be more expressive but note associate with an action. What do you think?